components/admin_console/reset_email_modal/reset_email_modal.tsx

```diff
@@ -157,7 +169,6 @@ export default class ResetEmailModal extends React.PureComponent {
                             onClick={this.doSubmit}
                             type='submit'
                             className='btn btn-primary'
-                            data-testid='resetEmailButton'
```

Comment:
Why removing this? It could potentially break E2E tests since thats what these are used for.

Reply:
I already add data-testid again on reset_email_modal.tsx, @hahmadia .